MaterialLib/Fluid/GibbsFreeEnergy/DimensionLessGibbsFreeEnergyRegion1.h

```diff
@@ -0,0 +1,102 @@
+/**
+ *  \brief Declare a class for dimensionless Gibbs free energy, region1.
+ *
+ *  \copyright
+ *   Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *              Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ *  \file   DimensionLessGibbsFreeEnergyRegion1.h
+ *
+ * Created on December 8, 2016, 12:31 PM
+ */
+
+#ifndef OGS_DIMENSION_LESS_GIBBS_FREE_ENERGY_REGION1_H
+#define OGS_DIMENSION_LESS_GIBBS_FREE_ENERGY_REGION1_H
+
+namespace MaterialLib
+{
+namespace Fluid
+{
+/**
+ *  A class for dimensionless Gibbs free energy defined by
+ *  \f[
+ *     \gamma=\sum_{i=1}^{34}\left[
+ *       n_i * (7.1-\pi)^{l_i}(\tau - 1.222)^{j_i}
+ *     \right]
+ *  \f]
+ * <a href="http://www.iapws.org/relguide/IF97-Rev.pdf"> IF97</a> 
```

Comment:
@bilke  and @endJunction Thanks. I thought it is in another class by checking the last commit only.